spring-mvc-simple/src/main/java/com/baeldung/spring/configuration/JadeTemplateConfiguration.java

```diff
@@ -16,24 +16,24 @@
 public class JadeTemplateConfiguration {
```

Comment:
The changes to this file and a couple of others below seem to be only spacing changes -- changing indents from tabs to spaces. Generally this is not advised, since it makes the change seem bigger than it actually is, and also because the reviewer has more files and lines of code to review.